build/variables.yml

```diff
@@ -61,7 +61,7 @@
   ProjectName: FlutterApplicationTemplate
 
   # Flutter version.
-  FlutterVersion: '3.22.1'
+  FlutterVersion: '3.27.3'
```

Comment:
3.29 was just released :D

Reply:
Yeah, I saw that :)
Spoke with @jeanplevesque for now, we'll stay with 3.27.3